### ISSUE TITLE
WARNING  Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.

### ISSUE THREAD

**Issue by elvisjade** [bootphon/shennong#7]
This error was reported when I was using this code, would like to ask how it will affect the result?

processor = CrepePitchProcessor(
   model_capacity='tiny', frame_shift=0.01)
pitch = processor.process(audio)

Colocations handled automatically by placer.
WARNING:tensorflow:From /home/jade/app/anaconda/envs/shennong/lib/python3.7/site-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python. ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.

**Comment by mmmaat** [comment]
Hi, thanks for reporting that. I just made a [PR](https://github.com/bootphon/shennong/pull/8) fixing that.
